.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+public/build/*
```

Comment:
excludes the transpiled javascript files. They should generate when you run the server the first time.